spec/views/insured/families/_qles_carousel.html.erb_spec.rb

```diff
@@ -53,27 +53,27 @@
   end
 
   context "QLE buttons for person with both roles" do
-    let(:person) { FactoryGirl.build(:person)}
-    let(:employee_role) { double }
-
     before :each do
       assign(:multiroles, true)
-      assign(:person, person)
-      allow(person).to receive(:active_employee_roles).and_return([employee_role])
-      allow(view).to receive(:individual_market_is_enabled?).and_return(false)
-      render "insured/families/qles_carousel"
     end
 
     it "contain buttons group for QLE roles" do
+      render "insured/families/qles_carousel"
       expect(rendered).to have_selector('div.market-selection')
     end
 
-    it "should have Employee  button for person with multiple roles" do
-      expect(rendered).to match(/Employee/)
+    it "should have Employee button for person with multiple roles" do
+      allow(view).to receive(:individual_market_is_enabled?).and_return(true)
+      render "insured/families/qles_carousel"
+      expect(rendered).to have_link('Employee')
+      expect(rendered).to have_link('Individual')
     end
 
-     it "should not have Individual button for person with multiple roles" do
-       expect(rendered).not_to match(/Individual/)
+    it "should have Individual button for person with multiple roles" do
```

Comment:
this should read something like it "should not display Individual button if individual market is disabled"